app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt

```diff
@@ -493,11 +505,27 @@ class EditSectionActivity : BaseActivity(), ThemeChooserDialog.Callback {
     fun clickNextButton() {
         when {
             editSummaryFragment.isActive -> {
+                if (invokeSource == Constants.InvokeSource.EDIT_ADD_IMAGE) {
+                    val pageTitle = intent.getParcelableExtra<PageTitle>(InsertMediaActivity.EXTRA_IMAGE_TITLE)
+                    ImageRecommendationsEvent.logAction("editsummary_save", "editsummary_dialog", ImageRecommendationsEvent.getActionDataString(
+                        filename = pageTitle?.prefixedText!!, recommendationSource = pageTitle.wikiSite.languageCode, recommendationSourceProject = pageTitle.wikiSite.languageCode,
+                        acceptanceState = "accepted", seriesNumber = "", totalSuggestions = "",
```

Comment:
It's not necessary to pass parameters that are empty (i.e. the same as their default value).